### PR TITLE
clean out bad ampersand in code

### PR DESCRIPTION
**Pull request by jacklatourette**
Fix issue with unescaped & in xml doc being used

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+attrs==18.2.0
```
Comment (seperman) [inline requirements.txt]: @jacklatourette What is attrs doing here?

Comment (jacklatourette) [reply requirements.txt]: attrs needed to be upgraded to be compatible the pytest version being used in circle ci

Comment (seperman) [reply requirements.txt]: @jacklatourette Ok then please move it to `requirements-dev.txt` where dev related packages shold reside.

File: modelmapper/cleaner.py [hunk 2]
```diff
@@ -157,13 +157,16 @@ def _mark_booleans(item):
                     try:
                         item = Decimal(item)
                     except Exception as e:
-                        raise TypeError(f'Unable to convert {item} into decimal: {e}') from None
+                        if self.settings.skip_failed_cast and is_nullable:
```
Comment (jacklatourette) [inline modelmapper/cleaner.py]: Should be allowed to skip casting for all types not just Decimal

**Comment by seperman** [comment]
Closing this PR since the issues that this PR was trying to address are addressed in a separate PR.

**Pull request closed without merging**